### PR TITLE
Harmonize Docker's Compose files

### PR DESCRIPTION
**Pull request by sirtawast**
## Description :sparkles:

I've decided to cherry-pick this one out of NextJS v12 PR. 

This enhancement makes it easier to swap branches, builds and change between projects / applications.

I've already demonstrated how this runs in today's (2023-04-17) little session so there's little to be approved.

- Add an app name prefix to scope each services so that docker can manage projects without deleting the existing one when switching from one app context to another
- Kesäseteli `docker-compose.youth.yml` is not needed as `handler` is more extended version from it and browser-tests fail if both the **handler AND youth** are not up.

image

### CODE REVIEW

**Review by arpargo — CHANGES_REQUESTED** [review]

File: docker-compose.tet-admin.yml [hunk 1]
```diff
@@ -28,10 +29,10 @@ services:
       - ./backend/shared:/shared
       - ./backend/tet/media:/var/media
     depends_on:
-      - postgres
-    container_name: te-bknd
+      - te-postgres
```
[the remark this answers is not in the document]
Comment (sirtawast) [reply docker-compose.tet-admin.yml:32]: Good catch! I will fix it tomorrow.

**Review by karisal-anders — APPROVED** [review]
Things that could be improved but if you are in a hurry you can just merge as is:
 - Remove everything mentioning use of `docker-compose` as it doesn't work anymore with the current docker-compose YAML files (You can try it and it blows up), `docker compose` seems to be required.
  - Having a double prefix is probably not needed for pgdata, e.g. `yjdh-kesaseteli_ks-pgdata` is probably spurious, it could just be `yjdh-kesaseteli_pgdata`

**Comment by sirtawast** [comment]
@arpargo could you re-test please 🙏

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://kesaseteli-pr1981.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://kesaseteli-pr1981.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**EMPLOYER branch is deployed to platta: https://kesaseteli-pr1981.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://handler-pr1981.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://handler-pr1981.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-pr1981.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**YOUTH branch is deployed to platta: https://youth-pr1981.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://youth-pr1981.dev.hel.ninja :laughing::tada::tada::tada:

**Review by arpargo — APPROVED** [review]
LGTM

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://kesaseteli-pr1981.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**YOUTH branch is deployed to platta: https://youth-pr1981.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**EMPLOYER branch is deployed to platta: https://kesaseteli-pr1981.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://youth-pr1981.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://handler-pr1981.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-pr1981.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://handler-pr1981.dev.hel.ninja :laughing::tada::tada::tada:

**Review by karisal-anders — APPROVED** [review]

**Pull request merged**